ci/build/upload_build_info.sh

```diff
@@ -6,9 +6,10 @@ set -ex
 readonly PIPELINE_POSTMERGE="0189e759-8c96-4302-b6b5-b4274406bf89"
 readonly PIPELINE_CIV1_BRANCH="0183465b-c6fb-479b-8577-4cfd743b545d"
 if [[
-  "${BUILDKITE_PIPELINE_ID:-}" != "${PIPELINE_POSTMERGE}" && 
-  "${BUILDKITE_PIPELINE_ID:-}" != "${PIPELINE_CIV1_BRANCH}" &&
-  "${BUILDKITE_BRANCH:-}" != "master"
+  (
+    "${BUILDKITE_PIPELINE_ID:-}" != "${PIPELINE_POSTMERGE}" && 
+    "${BUILDKITE_PIPELINE_ID:-}" != "${PIPELINE_CIV1_BRANCH}" 
+  ) || "${BUILDKITE_BRANCH:-}" != "master"
```

Comment:
maybe just split pipeline id from branch checking into 2 different `if` lines? so that the message is accurate?